style.css

```diff
@@ -0,0 +1,1453 @@
+@import url("https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap");
+
```

Comment:
add * { box-sizing: border-box }. so your half styling problems will be automatically solved.